packages/grid/x-data-grid-premium/src/components/GridColumnMenuRowUngroupItem.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import MenuItem from '@mui/material/MenuItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import {
+  gridColumnLookupSelector,
+  useGridSelector,
+  GridColumnMenuItemProps,
+} from '@mui/x-data-grid-pro';
+import { useGridApiContext } from '../hooks/utils/useGridApiContext';
+import { gridRowGroupingSanitizedModelSelector } from '../hooks/features/rowGrouping/gridRowGroupingSelector';
+import { useGridRootProps } from '../hooks/utils/useGridRootProps';
+
+function GridColumnMenuRowUngroupItem(props: GridColumnMenuItemProps) {
```

Comment:
Did you consider merging `GridColumnMenuRowUngroupItem` and `GridColumnMenuRowGroupItem` into a single component? It's kind of hard to understand why we have 2 components for a single `ColumnMenuGroupingItem` slot.
The quicker solution to this would be to export `GroupingItem` component instead of `GridColumnMenuRowUngroupItem` and `GridColumnMenuRowGroupItem`

Reply:
Makes sense, it's not very useful for the users to render only one of them as the functionality will be incomplete. Exported the wrapper.